DataFormats/BTauReco/interface/TemplatedSecondaryVertexTagInfo.h

```diff
@@ -16,7 +16,9 @@
 #include "DataFormats/BTauReco/interface/CandIPTagInfo.h"
 #include "DataFormats/Candidate/interface/VertexCompositePtrCandidate.h"
 #include <functional>
+#ifndef __APPLE__
 #include <ext/functional>
+#endif
```

Comment:
We removed `ext/functional` and the code still compiles. I bet someone included that when we were using C++11 in order to get some future C++14 code. Now that we are using C++14 the use of the header appears to be unnecessary.